src/utilities/utilities.js

```diff
@@ -216,6 +216,185 @@ export function body_surface_area(weight, height, formula = "dubois") {
     return 0.244 * Math.pow(weight, 0.383) * Math.pow(height, 0.693);
 
   throw new Error(
-    `This ${formula} to calculate the body_surface_area does not exists.`
+    `This ${formula} to calculate the body_surface_area does not exists.`,
   );
 }
+
+/**
+ * Estimates the relative air speed which combines the average air speed of the
+ * space plus the relative air speed caused by the body movement. Vag is assumed
+ * to be 0 for metabolic rates equal and lower than 1 met and otherwise equal to
+ * Vag = 0.3 (M - 1) (m/s)
+ *
+ * @template {(number | number[])} T
+ * @param {T} v - air spped measured by the sensor, [m/s]
```

Comment:
There are typos in this line and T is not the right variable name for airSpeed, see my previous comment and let's discuss this during our next meeting

Reply:
`T` here is not the variable name, it is the type of the provided parameter. `T` is defined by the `@template` tag on line 229.

Reply:
I am very sorry about that. Would it be possible to give this template a more meaningful and informative name?

Reply:
While it is possible to give the template any name you'd please, `T` is the accepted industry standard. Historically this is done to ensure that a compile time name is not confused with, for example, a parameter or variable name.